src/utils/config-utils.ts

```diff
@@ -20,8 +20,17 @@ export function createConfigFromEnv(env: ProcessEnv): Config {
   if (env['PORT']) {
     retVal.port = parseInt(env['PORT']);
   }
-  if (env['DB_URL']) {
-    retVal.dbUrl = env['DB_URL'];
+  if (env['DB_ENDPOINT']) {
+    retVal.dbEndpoint = env['DB_ENDPOINT'];
+  }
+  if (env['DB_URI_QUERY']) {
+    retVal.dbUriQuery = env['DB_URI_QUERY'];
+  }
+  if (env['DB_USER']) {
+    retVal.dbUser = env['DB_USER'];
+  }
+  if (env['DB_PASSWORD']) {
+    retVal.dbPassword = env['DB_PASSWORD'];
   }
   if (env['DB_SSL_VALIDATE']) {
```

Comment:
As noted in the other PR, if this is set we should automatically add the `ssl=true` to the query params I think, but happy for that to be raised as an issue and parked until post MVP.

Reply:
This change requires the types for the config-manager to be changed which could have side effects so I agree let's leave to post-MVP.